### DISCUSSION TITLE
EL9 aarch64 version of choria

### DISCUSSION THREAD

**Discussion by pedmon** [choria-io/general#55] [General]
I see that x86_64 and ppc64le are available on the choria repo but not one for aarch64. Can that be added? ARM definitely warrants addition. See: https://yum.us.choria.io/release/el/9/

**Comment by valentino-aguiar-gsa** [comment]
Hi, definitely agree. We've just started deploying AWS graviton instances.

**Comment by ripienaar** [comment]
Those who need this should contribute the needed packaging work in `go-choria/packager` and the actions.
